data/human/free worlds 1 start.txt

```diff
@@ -980,6 +980,19 @@ mission "FW Pirates 1"
 			
 			label agree
 			`	"So, here's the plan," he says. "These four systems belong to the pirates." He points to them on the map. "We want you to scout out all four systems. Just a quick fly-through, no need to land on their planets. Then report back to us, and we'll decide what our next step will be."`
+			choice
+				`	"Sounds good."`
+					goto leave
+				`	"What about the pirate systems to the north of Men?"`
+			`	Tomek shrugs. "Simply put, those worlds aren't a threat to us right now. From what we can gather, most of the pirate factions there are focused on internal power struggles at the moment rather than piracy in the Rim. Those CCOR guerrillas are more than enough to keep any pirate warlords there busy for the immediate future."`
+			choice
+				`	"CCOR?"`
+					goto ccor
+				`	"Understood."`
+					goto leave
+			label ccor
+			`	Tomek looks surprised. "I assumed you knew. The Confederated Councils of the Outer Rim was a government that sought to practice its own particular flavor of communism outside of the Republic's influence. They fell around 50 years ago, and their former planets have since been under the control of a mixture of pirate gangs and guerrillas looking to restore the CCOR. I wouldn't worry about the latter too much either - at this point they're little more than a pirate gang themselves."`
```

Comment:
This seems to be all grammatically correct, but it feels a little stilted. "Sought" sounds a little bit snobby to me in this context, even though it works just fine in other places, and "their former planets have since been under the control of a mixture of pirate gangs and guerrillas looking to restore the CCOR" seems a bit run-on-y.

Reply:
I agree with the note on 'sought' but I'm not sure on replacement given it's Tomek speaking. IIRC he's the more hot-headed FW figure, right? The one always advocating force verses the pirates.